lastadapter/src/main/java/com/github/nitrico/lastadapter/Holder.kt

```diff
@@ -19,6 +19,6 @@ package com.github.nitrico.lastadapter
 import android.databinding.ViewDataBinding
 import android.support.v7.widget.RecyclerView
 
-open class Holder<B : ViewDataBinding>(val binding: B) : RecyclerView.ViewHolder(binding.root) {
+open class Holder<B : ViewDataBinding>(val binding: B?) : RecyclerView.ViewHolder(binding?.root) {
```

Comment:
the nullable binding variable will break a lot of existing code. You can/should even remove it as DataBindingUtil.inflate doesn't return a nullable binding (anymore?)

Reply:
They should've reverted this change. I don't have the link handy but in one of the issue in issue tracker Yigit Boyar mentioned that Databinding is supposed to be null in case the layout is not binding layout. And this was Working as intended type of thing,

At this point

1. I am awaiting official confirmation in release notes
2. At work I can still see Databinding.inflate() retirns Binding! class. Which can be null from Java if not from kotlin.
3.  This comment increases confusion a lot. Coming from `DataBindingUtil.java` 3.1.0beta1.
``` java
// @Nullable don't annotate with Nullable. It is unlikely to be null and makes using it from
    // kotlin really ugly. We cannot make it NonNull w/o breaking backward compatibility.

``` 


I will wait for official release notes around databinding changes before deciding to proceed on this change.

Reply:
Sure, makes sense!

Even if it'll be nullable we might work around that by checking for null before creating the Holder-instance and optionally throw an exception if it the binding is null. That way we wouldn't break backward compatibility with this library.

Reply:
>  optionally throw an exception

I am against throwing an exception in this case.

Reason(s)

1. Mostly this lib is being used with normal databinding, which means users have to handle binding nullability in places where this library is not used. 

2. Most thrown exception is swallowed without having any actions. Based on this I fear throwing exception won't change anything except adding try catch block, mostly to log exceptions. (Even we do this in 80% cases)

Anyways, I don't mind if the flow goes either way (Having nullable bindings or throwing exception),

Reply:
I agree. Let's see - maybe the google devs decide to handle this case (the user tries to load a non-bindable layout) themselves to sort this out.

Reply:
Update on null-ability on databinding, Here is the [issue we reported w.r.t. nullability](https://issuetracker.google.com/issues/70908732), based on their response what they did is not to force null-ability when we're using kotlin. I don't know how this changes our discussion but I am putting this as reference link.